index.html

```diff
@@ -12,16 +12,22 @@
 <body>
 <section class="topsection">
   <div class="container topsection__container">
-    <div class="topsection__header">
-      <div class="topsection__logo"></div>
-      <div class="topsection__header__items">
-        <ul class="topsection__header__links">
-          <li class="topsection__header__link active">Stays</li>
-          <li class="topsection__header__link">Attractions</li>
+    <div class="header">
+      <svg class="header__logo">
```

Comment:
Block is a functionally independent page component that can be reused. Can we reuse the block header anywhere else on the page?